### DISCUSSION TITLE
[HA] High Availability - Kubernetes

### DISCUSSION THREAD

**Discussion by joaopedrocg27** [immich-app/immich#9504] [Q&A] [answered]
Is it possible to run Immich in HA? Like multiple pods of microservices/server/ML? I tried to look but I found no docs regarding this.

I have some downtimes when pods restart :)

**Comment by bo0tzz** [comment, marked as answer]
Yes

**Reply by joaopedrocg27** [reply]
Would it be good to add this replica count to the chart?

**Reply by bo0tzz** [reply]
It's already there https://github.com/bo0tzz/kube/blob/main/apps/immich/immich.yaml#L87
